spec/knapsack_pro/base_allocator_builder_spec.rb

```diff
@@ -154,7 +154,7 @@
       end
 
       context 'when rake task to detect RSpec test examples works' do
-        let(:slow_test_files) { double(size: 5) }
+        let(:slow_test_files) { double(size: 5, empty?: false) }
```

Comment:
Is `slow_test_files` an array? If so, I'd pass a real thing rather than a stub.

Reply:
Ok. I use a real array now.